packages/lodestar/src/api/impl/beacon/state/utils.ts

```diff
@@ -12,6 +12,9 @@ import {StateContextCache} from "../../../../chain/stateCache";
 import {IBeaconDb} from "../../../../db";
 import {getStateValidatorIndex} from "../../utils";
 import {StateId} from "./interface";
+import {sleep, assert} from "@chainsafe/lodestar-utils";
+import {processSlots} from "@chainsafe/lodestar-beacon-state-transition/lib/phase0/fast";
```

Comment:
Can you do `import {phase0} from "@chainsafe/lodestar-beacon-state-transition";`? And below: `phase0.fast.processSlots(...)`
We're trying to keep all of our phase-specific code explicitly namespaced where its used.